src/plots/gl3d/layout/axis_attributes.js

```diff
@@ -14,6 +14,15 @@ var extendFlat = require('../../../lib/extend').extendFlat;
 
 
 module.exports = {
+    visible: {
+        valType: 'boolean',
+        role: 'info',
+        description: [
```

Comment:
Could reuse the declaration from `cartesian/` with `visible: axesAttrs.visible,`